### DISCUSSION TITLE
Build and Deploy job error

### DISCUSSION THREAD

**Discussion by solarsystemsit** [KelvinTegelaar/CIPP#2042] [Q&A]
Hi,

I've just tried troubleshooting this myself but I can't work out what's causing it. I'm getting this error. Has anyone seen this before?

I've had my CIPP instance working for months and then all of a sudden this erorr pops up.

![image](https://github.com/KelvinTegelaar/CIPP/assets/144329096/244618a0-e282-4ba7-a6e7-259eeb25ced6)

**Comment by KelvinTegelaar** [comment]
This is a support question - Only sponsors get can support.

**Reply by solarsystemsit** [reply]
Thanks
